mesh/mesh.cpp

```diff
@@ -9674,6 +9674,9 @@ GeometricFactors::GeometricFactors(const Mesh *mesh, const IntegrationRule &ir,
    const int NQ   = ir.GetNPoints();
 
    Vector Enodes(vdim*ND*NE);
+
+   //Support for DG space missing
+
```

Comment:
Unless I'm mistaken, I think this was addressed already by #1156, so there's no need to add this comment.

Reply:
👍